app/src/main/res/values-de/strings.xml

```diff
@@ -139,4 +139,5 @@
   <string name="copyAddress">Kopieren</string>
   <string name="txReceivedOn">Empfangsadresse</string>
   <string name="twoFacGauthTextCode">Oder sie benutzen den folgenden Code: %s. Vergessen sie nicht in zu sichern!</string>
+  <string name="sendPay"></string>
```

Comment:
I'm guessing this shouldn't be blank? Perhaps remove if you don't have the translation yet?